mlir/test/Conversion/TensorToLinalg/tensor-ops-to-linalg.mlir

```diff
@@ -44,3 +46,26 @@ func.func @generalize_pad_tensor_dynamic_shape(%arg0: tensor<4x?x2x?xf32>, %arg1
   } : tensor<4x?x2x?xf32> to tensor<4x?x?x?xf32>
   return %out : tensor<4x?x?x?xf32>
 }
+
+// -----
+
+// Ensure that the constant value inside the PadOp block does not cause a crash.
```

Comment:
[nit] Every test makes sure that the edge case being exercised doesn't crash. To me it would be more helpful to add some info that's _unique_ to this test. In particular, the fact that despite the pad value being constant, the "lowered" version is `tensor.generate` rather than `linalg.fill`. And that that's because the constant is defined in the PadOp region. That's clear _today_, but won't be so obvious in the future. My suggestion - update the comment.

Reply:
Good idea.